src/libraries/Common/tests/TestUtilities/System/SecretsHelper.Net.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
Remove third line of license header. It's been updated globally in https://github.com/dotnet/runtime/pull/38793